### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by GilbertMiao**

### CODE REVIEW

**Comment by pzckwoop** [comment]
need to change something

**Comment by GilbertMiao** [comment]
ok i iwll change

File: README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
   1. This is for testing.
   2. new test
   3. A real good PR eg
+  4. test case: bad pr, commits after comments
```
Comment (pzckwoop) [inline README.md]: need to be changed

**Review by pzckwoop — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (GilbertMiao) [reply README.md]: ok

**Pull request merged**